api/files/serializers.py

```diff
@@ -395,7 +402,7 @@ class FileSerializer(BaseFileSerializer):
 
     cedar_metadata_records = RelationshipField(
         related_view='files:file-cedar-metadata-records-list',
-        related_view_kwargs={'file_id': '<_id>'},
+        related_view_kwargs={'file_id': 'get_file_guid_or_error'},
```

Comment:
For CR: ideally, we want to change `<file_id>` to `<file_guid>` in the URL and view too to avoid confusion